stl/inc/optional

```diff
@@ -418,7 +418,7 @@ public:
     _NODISCARD constexpr remove_cv_t<_Ty> value_or(_Ty2&& _Right) const& {
         static_assert(is_convertible_v<const _Ty&, remove_cv_t<_Ty>>,
             "The const overload of optional<T>::value_or requires const T& to be convertible to remove_cv_t<T> "
-            "([optional.observe]/18 as modified by LWG-XXXX).");
+            "(N4885 [optional.observe]/18 as modified by LWG-3424).");

```

Comment:
Pre-existing: from LWG-3424, the paragraph numbers in these citations are incorrect. They should be /17 and /19 as stated in the issue which agrees with N4885.

Reply:
Oops, I didn't check the paragraph numbers, or I would have mentioned them in the issue. My mistake, so I'll push changes to fix it. :smiley_cat: